app/views/projects/bots/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= render Viral::PageHeaderComponent.new(title: t('.title'), subtitle: t('.subtitle')) do |component| %>
+  <%= component.with_icon(name: "users", classes: "h-14 w-14 text-primary-700") %>
+  <%= component.with_buttons do %>
+    <%= link_to t(".add_new_bot"),
+    new_namespace_project_bot_path(@namespace.parent, @namespace.project),
+    data: {
+      turbo_frame: "bot_modal",
+      turbo_stream: true
+    },
+    class: "button button--state-primary button--size-default" %>
```

Comment:
**nitpick:** Can you update these buttons too

Reply:
Updated in 93d4124ba618febbc1536fc496f62a267f742847